File: DFRobot_RGBLCD/cpp/Blink/Blink.cpp
#include "../DFRobot_RGBLCD.h"
#include "stdio.h"

int main(){

  DFRobot_RGBLCD lcd(16, 2);
  lcd.init();

  lcd.print("Hello World!");
  delay(1000);
  
  while(1){
    printf("hello\r\n");
    //Turn off the blinking cursor:
    lcd.stopBlink();
    delay(3000);
    //Turn on the blinking cursor:
    lcd.blink();
    // printf("===");
    // delay(3000);
    delay(3000);
  }
  return 0;
}

File: DFRobot_RGBLCD/python/Blink/Blink.py
import sys
sys.path.append('../')
import rgb1602
import time

lcd = rgb1602.RGB1602(16,2)

lcd.printout("hello world!");
time.sleep(1)
while True:
  lcd.stopBlink()
  time.sleep(3)
  lcd.blink()
  time.sleep(3)
